include/schema/common/article.json

```diff
@@ -40,6 +40,12 @@
             "default": true,
             "nullable": true
         },
+        "updated_auto_hide": {
```

Comment:
How about replace `update_auto_hide` with `update_time` and allow `true`, `false`, `"auto"` as acceptable values?

- `true`: always show article update time when `page.updated` is set.
- `false`: always hide article update time.
- `"auto"`: show article update time only when `page.updated` is set and it is different from `page.date`.

Reply:
Good idea! :D